lib/index.js

```diff
@@ -83,21 +99,49 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
             public: true,
           });
       }
-      // TODO: call `brAccount.getAll`
-      // TODO: improve error details
-      throw new BedrockError(
-        'Account does not exist.', 'NotFoundError', {
-          httpStatusCode: 404,
-          public: true,
-        });
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {'account.email': email};
+      if(cursor) {
+        query['account.id'] = {$gt: cursor};
+      }
```

Comment:
I don't know how I feel about using the name `cursor` for this ... but I'm going to accept it for now. It feels like exposing an underlying implementation detail rather than just calling out what it is. Or perhaps it is the technical nature of the term. Maybe something like `after` or `last` is a better name.

Reply:
after I think is better actually it across the function of it better. so no issue. cursor is the official technical term for it I think.